src/components/RadioButton/RadioButton.driver.ts

```diff
@@ -8,18 +9,30 @@ import { RADIOBUTTON_DATA_KEYS } from './dataHooks';
 export interface RadioButtonDriver extends BaseUniDriver {
   isChecked(): Promise<boolean>;
   isDisabled(): Promise<boolean>;
+  isFocused(): Promise<boolean>;
+  clickInput(): Promise<void>;
+  blurInput(): Promise<void>;
 }
 
 export const radioButtonDriverFactory = (
   base: UniDriver,
 ): RadioButtonDriver => {
   return {
     ...baseUniDriverFactory(base),
+    async clickInput() {
+      return base.$('input').click();
+    },
+    async blurInput() {
+      return Simulate.blur(await base.$('input').getNative());
+    },
```

Comment:
Oh sorry for the ping-pong, I just noticed this.
We also can't use Simulate inside the driver.

The driver is a [Unidriver](https://github.com/wix-incubator/unidriver/blob/master/core/src/index.ts), and it should support various different environments, such as `enzyme`, `puppeteer`, `protractor`, etc.

So we can't use libraries that assume something on the tests environment.
In this case SImulate is used in environments where a dom is present (unlike `puppeteer` for instance).

So, we'll need to remove this method from the driver, and simulate blur in the test file itself.
At least until we'll have support for this via Unidriver.

Again, sorry I didn't notice this earlier